core/store/benches/finalize_bench.rs

```diff
@@ -36,7 +36,7 @@ use rand::prelude::SliceRandom;
 
 /// `ShardChunk` -> `StoreUpdate::insert_ser`.
 ///
-/// ~10ms or faster for ~24MB receipts
+/// ~6ms or faster for ~24MB receipts
```

Comment:
I guess this depend on hardware and other factors. Is that result from your machine or some gcp instance? Is there some standard specs that others can use to repro those values?

Reply:
See commit message for hardware specs. I assume people who really want to know will be able to use git blame.

But yeah, it won't be reproducible. Hm, I didn't really expect it to be necessary to reproduce. I merely wanted give a hint for what to expect and explain the relative performance between the benchmarks. When someone needs a good comparison, they should check out the different commits and run the same benchmark on the same machine. At least that my rational here. Hence why I committed the benchmark separately first.